src/text-transform/utils.ts

```diff
@@ -28,7 +28,12 @@ export const DEFAULT_ALLOWED_TAGS = [
 ];
 export const typografConfig = {
     enabled: ['common/nbsp/afterNumber', 'common/nbsp/afterParagraphMark'],
-    disabled: ['common/symbols/cf', 'ru/other/phone-number'],
+    disabled: [
+        'common/symbols/cf',
+        'ru/other/phone-number',
+        'common/space/afterColon',
+        'common/space/afterSemicolon',
```

Comment:
why do we remove the space after the semicolon and the formatting of the number?

Reply:
- after semicolon we have same problem, and content want to fix it
- it's history, maybe maintainers khow